bin/reth/src/commands/stage/unwind.rs

```diff
@@ -40,6 +40,10 @@ pub struct Command {
 
     #[command(subcommand)]
     command: Subcommands,
+
+    /// If this is enabled, then _only_ the offline stages (headers, bodies) will be unwound.
```

Comment:
is this comment correct? It should be "all except headers and bodies" I think

Reply:
oops yeah got it mixed up